src/components/Modals/CourseMenu.vue

```diff
@@ -216,7 +267,9 @@ export default defineComponent({
 
 <style scoped lang="scss">
 @import '@/assets/scss/_variables.scss';
-
+.delete-button .courseMenu-text {
+  color: #eb6d6d;
```

Comment:
Is there no color variable for this?

Reply:
Don't see one. Seems like this color is mentionned around 3 times in our code-base? I can make it into a `scss` variable.